_posts/2021-03-20-march-meetup.md

```diff
@@ -0,0 +1,58 @@
+
+  
+
+The first of the talks was by Kanishk who had previously presented a session at our meetup in November of 2020. Considering the response from that session and the number of queries that came up in the session then, Kanishk found the need to bridge this gap and so will be presenting a three-part workshop series in our meetups. This session aptly named "Machine Learning in action" was a precursor to what's in store in the upcoming workshop session.
+
+The session was intended to take the audience through the different solutions that ML has enabled us with. We were taken through what ML was and the goal of ML define as-"The goal is to make the guesses good enough to be useful and never to make them perfect guesses". Kanishk took us through the expanse of the types of ML out there with detailed use-case discussion for each. These types included:
+- Supervised Learning
+- Unsupervised Learning
+- Reinforcement Learning
+- Hybrid Learning
+- Statistical Learning
+
+We also got to know about the different learning techniques which ranged from Multi-Task Learning to Ensemble Learning with mentions of Active, Online Learning. The part of the session that put everyone in awe was when the applications of ML were demoed, these demos included:
+- Image classification through CIFAR-10 model
+- Application of GANs
+	- Grainy image correction.
+	- [Nvidia's GauGAN](http://nvidia-research-mingyuliu.com/gaugan/) interactive demo.
+	- The real person like images from [This person does not exist](https://thispersondoesnotexist.com/).
+	- Image to image translations with [pipx2pix model](https://phillipi.github.io/pix2pix/).
+	- Google's [quick draw](https://quickdraw.withgoogle.com/).
+	- Interactive sketch prediction.
+- Application of NLP through the use of GPT-2 APIs
+
+The demo packed talk session was followed by 2 lightning talks. The first of which was presented by Bangpyper's veteran member [Kracekumar](https://kracekumar.com/). He spoke about a tool that he built over a weekend to render Jupyter Notebook content on terminals called JUT. The need for this solution was stated as the inconvenience of always having to rely on browsers and editors to view them. The approach that was taken here to render notebooks on terminals was to use the JSON source of .ipynb files to parse using the available schema and display them on the terminal. Karce took us through the different technical hurdles that he had in building this solution and also listed the features he has planned to integrate. JUT is open for collaborations and suggestions, find the project here: [https://github.com/kracekumar/jut](https://github.com/kracekumar/jut)
```

Comment:
Typo in my name "karce took us..."

Reply:
Oops, my apologies @kracekumar...I have fixed this in PR #47.
Thanks for pointing this out.